uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Menu/Config/Configure.php

```diff
@@ -0,0 +1,303 @@
+<?php
+
+return [
```

Comment:
If I was a dev who just started working on the CiviHR project and my first task was to change something in the menu configuration file I would never think of trying to find this file inside the sources folder. The first thing I would probably do is to look for some "config" folder in the root of the application (the hrcore root folder in this case), as this is the common approach in most systems and frameworks. 

Since we have several files defining the menu, I would suggest placing them all in `hrcore/config/menu`. Also, since these are not classes anymore, there's no need to capitalize the file name.